BenchmarkVCFs/BenchmarkBoard/BenchmarkBoard-MS.ipynb

```diff
@@ -0,0 +1,695 @@
+{
```

Comment:
They're essentially identical (I think only differing in the `start_app` syntax at the end). The reason I included both was: the notebook version is amenable to doing some custom changes at the top and running, which I often like to do (e.g. adding another column to the data, etc). On the other hand, the script version is ready to go "out of the box" without modification, assuming the WDL outputs are in the same directory. So the former is better for manual editing, whereas the latter is more automated. I don't mind getting rid of one, but I'd prefer to keep the notebook since that's the way I usually like to run it. It's possible to convert from .ipynb to .py, but I think that requires Jupyter to be installed. Maybe it's not asking too much that everyone who'd use this script already has that anyway?